book/src/development/contributing.md

```diff
@@ -0,0 +1,116 @@
+# Contributing
+
+Halloy! If you don't know it yet, but "Halløj" is the a danish "Hello" or "Hi". Thank you for visiting our development guide.
+
+You have various options for contributing to Halloy. Before you start working on Halloy, read the topics “[Be nice](#be-nice)”, "[Goals](#goals)" and "[Licensing](#licensing)"
+
+- [Contributing](#contributing)
+  - [Goals](#goals)
+  - [Licensing](#licensing)
+  - [Contributing code](#contributing-code)
+    - [Coding Prerequisites](#coding-prerequisites)
+  - [Codebase Overview](#codebase-overview)
+    - [Project Structure](#project-structure)
+    - [Data Flow](#data-flow)
+  - [Coding-Standards \& Guidelines](#coding-standards--guidelines)
+    - [Formatting \& Linting](#formatting--linting)
+
+## Goals
+
+- **simple:** The UI and functions should be intuitive and accessible.
+- **fast:** *TBD*
+- **eternal:** Halloy will be the last actively used software at the end of time.
```

Comment:
The goal for Halloy is really just a simple, yet powerful multi-platform IRC client.